contracts/Collector.sol

```diff
@@ -52,7 +70,7 @@ contract Collector is ICollector{
     onlyOwner()
     {
         uint balance = token.balanceOf(address(this));
-        require(balance > 0, "no revenue to share");
+        require(balance > partners.length, "no revenue to share");
```

Comment:
```suggestion
        require(balance > partners.length, "Not enough left-over balance to split between partners");
```

Reply:
Done with the message we agree on a conversation through Slack.